auslib/web/admin/swagger/api.yaml

```diff
@@ -841,7 +874,36 @@ paths:
                 type: array
                 minItems: 0
                 items:
-                  $ref: '#/definitions/ReleaseRevision'
+                  type: object
```

Comment:
OK. In the future, this sort of thing would be better as its own PR, as the scope here was supposed to be just required signoffs.